### PR TITLE
crud update

### PR DESCRIPTION
**Pull request by charneff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,12 +64,14 @@ For this project, you must:
...
 logged in and the creator of that resource. For example, if we consider the
 example described below with models of User, DogHouse, and Review, I would only
-be able to edit or delete the reviews that I created. This protection should
-occur in the back end of the project. Simply altering the front end to hide the
-edit & delete buttons is insufficient in terms of security. Assuming you have a
-current user method and a post belongs to a user, the code needed to secure
-these operations looks something like this: `if current_user.id ==
-post.user.id`.
+be able to edit or delete the reviews that I created. This protection should occur
+in the backend of the project. Simply altering the front end to hide the edit & delete
```
Comment (lizbur10) [inline README.md]:
```suggestion
in the back end of the project. Simply altering the front end to hide the edit & delete 
```

File: README.md [hunk 2]
```diff
@@ -64,12 +64,14 @@ For this project, you must:
...
-be able to edit or delete the reviews that I created. This protection should
-occur in the back end of the project. Simply altering the front end to hide the
-edit & delete buttons is insufficient in terms of security. Assuming you have a
-current user method and a post belongs to a user, the code needed to secure
-these operations looks something like this: `if current_user.id ==
-post.user.id`.
+be able to edit or delete the reviews that I created. This protection should occur
+in the backend of the project. Simply altering the front end to hide the edit & delete 
+buttons is insufficient in terms of security. The code needed to secure these operations
+looks something like this `if current_user.id == post.user.id` assuming you have a
```
Comment (lizbur10) [inline README.md]:
```suggestion
looks something like this: `if current_user.id == post.user.id`.
```

File: README.md [hunk 3]
```diff
@@ -64,12 +64,14 @@ For this project, you must:
...
 example described below with models of User, DogHouse, and Review, I would only
-be able to edit or delete the reviews that I created. This protection should
-occur in the back end of the project. Simply altering the front end to hide the
-edit & delete buttons is insufficient in terms of security. Assuming you have a
-current user method and a post belongs to a user, the code needed to secure
-these operations looks something like this: `if current_user.id ==
-post.user.id`.
+be able to edit or delete the reviews that I created. This protection should occur
+in the backend of the project. Simply altering the front end to hide the edit & delete 
+buttons is insufficient in terms of security. The code needed to secure these operations
```
Comment (lizbur10) [inline README.md]:
```suggestion
buttons is insufficient in terms of security. Assuming you have a `current_user` method and a post belongs to a user, the code needed to secure these operations
```

File: README.md [hunk 4]
```diff
@@ -64,12 +64,14 @@ For this project, you must:
...
+looks something like this `if current_user.id == post.user.id` assuming you have a
+current user method and a post belongs to a user. Alternatively, the most performant way
```
Comment (lizbur10) [inline README.md]:
```suggestion
Alternatively, the most performant way
```

File: README.md [hunk 5]
```diff
@@ -64,12 +64,14 @@ For this project, you must:
...
+to implement this is `post = current_user.posts.find(params[:id])` with an if statement
```
Comment (lizbur10) [inline README.md]:
```suggestion
to implement this is:
```

File: README.md [hunk 6]
```diff
@@ -64,12 +64,14 @@ For this project, you must:
...
+starting on the next line `if post` do something, `else` do something else.
```
Comment (lizbur10) [inline README.md]:
```suggestion

```

File: README.md [hunk 7]
```diff
@@ -64,12 +64,14 @@ For this project, you must:
...
+
```
Comment (lizbur10) [inline README.md:84]:
```ruby
 post = current_user.posts.find(params[:id])
if post
  <do something>
else
  <do something else>
end
```

**Review by lizbur10 — CHANGES_REQUESTED** [review]
Hi Charlotte! Sorry this slipped through the cracks earlier :)

Doing the line by line suggestions got a little unwieldy so I might have screwed something up. Please check it!

File: README.md (hunk 7 above)
Comment (charneff) [reply README.md:84]: I think I got this added properly ;)

**Review by lizbur10 — APPROVED** [review]
Looks good!

**Pull request merged**